### PR TITLE
Fixes cargo investigate log spam

### LINKED ISSUE #6592: Cargo investigate log full of rubbish
```
13:06 [0x2008644] (89,119,1) || the supply shuttle console Remobothen has sent 
the supply shuttle away. Remaining points: 52. Shuttle contents: propulsion propulsion 
propulsion propulsion propulsion heater grille shuttle window heater grille shuttle window 
heater grille shuttle window medical crate:. crate:. freezer:. crate:. conveyor belt 
supply dock loading door crate: supply manifest. Supply Shuttle Airlock crate:. 
Loading Doors Loading Doors internals crate:. crate:. Supply Shuttle Airlock crate:. 
conveyor belt supply dock loading door insulated gloves crate :..
```

### PR DESCRIPTION
**Pull request by tkdrg**
Only non-anchored AMs will be displayed now. Empty crates will also get
a `(empty)` message instead of the weird `:.`.

Fixes #6592

### CODE REVIEW

**Comment by tkdrg** [comment]
Conflicts fixed.

**Comment by tkdrg** [comment]
This has been merge-ready for a while.

**Comment by Carn** [comment]
@aranclanos

File: code/controllers/subsystem/shuttles/supply.dm [hunk 1]
```diff
@@ -80,12 +80,14 @@
...
 	for(var/atom/movable/MA in areaInstance)
-		SSshuttle.sold_atoms += " [MA.name]"
 		if(MA.anchored)	continue
+		SSshuttle.sold_atoms += " [MA.name]"
 
 		// Must be in a crate (or a critter crate)!
 		if(istype(MA,/obj/structure/closet/crate) || istype(MA,/obj/structure/closet/critter))
 			SSshuttle.sold_atoms += ":"
+			if(!MA.contents)
+				SSshuttle.sold_atoms += " (empty)"
```
Comment (phil235) [inline code/controllers/subsystem/shuttles/supply.dm]: MA.contents never returns false even when the crate is empty, which means " (empty)" is never added.

**Comment by tkdrg** [comment]
Should be fixed.

**Comment by phil235** [comment]
Also, on line 380, `"[usr.key] has sent the supply shuttle away. Remaining points: [SSshuttle.points]. Shuttle contents:[SSshuttle.sold_atoms]."` The period after [SSshuttle.sold_atoms] isn't needed.

**Comment by tkdrg** [comment]
Fixed and squashed.

**Pull request merged**